omod/src/test/java/org/openmrs/module/referenceapplication/page/controller/LoginPageControllerTest.java

```diff
@@ -93,7 +93,9 @@
 	private static final String VIEW_LOCATIONS = "View Locations";
 	
 	private final UiUtils uiUtils = new UiUtils() {
-		
+		@Override
+		public boolean convertTimezones() {return  false;}
```

Comment:
What happens when we do not override this?

Reply:
If we don't override it, the test `post_shouldAutoSelectALocationAfterAuthenticationIfLoginLocationsSizeIsOne` will fail if the GP that handle timezones are true. 
Lets imagine that the GP that handle timezones is true, and we only have 1 location, if we don't have the property `clientTimezone` on the DB for that user, then, it will force the user to select one location when he do the login, even if there is only 1 location to select and that is why that test fail, in a scenario without timezones, if we only have 1 location, the user will skip the location selection page.